package.json

```diff
@@ -20,9 +20,9 @@
     "angular-route": "1.5.11",
     "angular-sanitize": "1.5.11",
     "angular-toarrayfilter": "1.0.2",
-    "angular-ui-bootstrap": "1.3.3",
     "angular-ui-router": "0.4.3",
-    "bootstrap": "3.3.7",
+    "angular1-ui-bootstrap4": "^2.4.22",
```

Comment:
For consistency with the other entries (and to make it easier to see what needs updating, given lack of tools compatibility with reading yarn.lock for outdatedness), I think we should remove the `^`.